components/LandingPage/LandingPageHero.tsx

```diff
@@ -0,0 +1,43 @@
+import { Image } from 'components/Image';
+import styles from '../../styles/LandingPageHero.module.scss';
+import { ImageTypes } from '../../types/Base';
+
+type LandingPageHeroProps = {
+    image: ImageTypes;
+    children: React.ReactNode;
+};
+
+export const LandingPageHero = ({ image, children }: LandingPageHeroProps) => {
+    return (
+        <>
+            <svg width="0" height="0" viewBox="0 0 1 1">
+                <defs>
+                    <clipPath
+                        id="shardClipPath"
+                        clipPathUnits="objectBoundingBox"
```

Comment:
question: what does this do?

Reply:
This sets the clip path to be relative to the object that is being clipped, on a scale of 0 to 1. If we used the other setting, `userSpaceOnUse`, we'd be clipping according to the co-ord system of the svg viewbox of the `<div>`, which is non-existent. [MDN docs](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/clipPathUnits).

Reply:
I've tested this across browsers with Browserstack and removed the unnecessary viewBox and width & height definitions